.erda/migrations/pipeline/20220117-pipeline-definition-add-index.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE `pipeline_definition` ADD INDEX `idx_category` (`category`);
+ALTER TABLE `pipeline_definition` ADD INDEX `idx_executor` (`executor`);
+ALTER TABLE `pipeline_definition` ADD INDEX `idx_creator` (`creator`);
+ALTER TABLE `pipeline_definition` ADD INDEX `idx_status` (`status`);
```

Comment:
what sql you use, should we combine to one index?

Reply:
done